cmd/nerdctl/commit.go

```diff
@@ -89,27 +79,34 @@ func commitAction(clicontext *cli.Context) error {
 	return nil
 }
 
-func newCommitOpts(clicontext *cli.Context) (*commit.Opts, error) {
-	rawRef := clicontext.Args().Get(1)
+func newCommitOpts(cmd *cobra.Command, args []string) (*commit.Opts, error) {
+	rawRef := args[0]
```

Comment:
seems different from the previous code

Reply:
It's same, because args from cobra start with 0, and doesn't include command name.